forms-flow-theme/scss/_modal.scss

```diff
@@ -520,4 +520,16 @@
     gap: var(--spacer-100);
     align-self: stretch;
   }
-}
\ No newline at end of file
+}
+
+.newversion-modal-body {
+  padding: 2rem 2.5rem !important;
+  display: flex;
+  flex-direction: column;
+  gap: var(--Rem1_5, 24px);
```

Comment:
this is wrong. please remove default value and use correct variable @Josephalexantony-aot